wardrobe/timestamp/src/lib.rs

```diff
@@ -249,7 +237,8 @@ impl<V: Verifier + From<UpForGrabs>, T: TimestampConfig + 'static> TuxedoInheren
 
         let new_output = Output {
             payload: new_timestamp.into(),
-            verifier: UpForGrabs.into(),
+            verifier: V::new_unspendable()
+                .expect("Must be able to create unspendable verifier to use timestamp inherent."),
```

Comment:
This is one panic I'm expecting to see when I run this node. But I don't see it.